data/mwaa/requirements/requirements-eks-operator.txt

```diff
@@ -0,0 +1 @@
+airflow-kubernetes-job-operator~=2.0.14
```

Comment:
Does this work with the specified airflow version (2.2.2)?

Reply:
Do we still need this file, since mwaa is gone?

Reply:
No we don't I removed it